pkg/relayer/session/claim.go

```diff
@@ -170,11 +162,20 @@ func (rs *relayerSessionsManager) waitForEarliestCreateClaimsHeight(
 	logger.Info().Msg("waiting & blocking until the earliest claim commit height for this supplier")
 
 	// Wait for the earliestSupplierClaimsCommitHeight to be reached before proceeding.
-	_ = rs.waitForBlock(ctx, earliestSupplierClaimsCommitHeight)
-
-	logger.Info().Msg("observed earliest claim commit height")
+	blockObserved := make(chan struct{}, 1)
+	go func() {
+		_ = rs.waitForBlock(ctx, earliestSupplierClaimsCommitHeight)
+		logger.Info().Msg("observed earliest claim commit height")
```

Comment:
Not 100% sure about the syntax but let's add some detail:

```suggestion
		logger.Info().Msgf("observed earliest claim commit height while waiting for block %d", earliestSupplierClaimsCommitHeight)
```